### PR TITLE
Change escaping of percentage sign (DEV)

### PR DESCRIPTION
**Pull request by ralfgehrer**
All other options did not work well with the internal translation tool.

TEST:
- Check the statistics info screen.
- Scroll to Legend
- See Trend
- Check the description and make sure it says "...1%..."  and "...5%..."

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt [hunk 1]
```diff
@@ -28,6 +28,12 @@ class StatisticsExplanationFragment : Fragment(R.layout.fragment_statistics_expl
             R.string.statistics_explanation_seven_day_r_link_label,
             R.string.statistics_explanation_faq_url
         )
+
+        binding.statisticsExplanationTrendText.apply {
+            val label = String.format(getString(R.string.statistics_explanation_trend_text))
+            text = label
+            contentDescription = label
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt:35]: You just adapted what was there in XML, but why do we even need `contentDescription` on a `TextView` without any difference to the text?.

**Review by harambasicluka — APPROVED** [review]
Finally! :)
IMO the `contentDescription` can be removed.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt (hunk 1 above)
Comment (harambasicluka) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt:35]: Is it due to?
> Check the description and make sure it says "...1%..." and "...5%..."

Comment (ralfgehrer) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt:35]: In this case, yes.

**Comment by axelherbstreith** [comment]
Looking good on Samsung S8 but somehow the app crashes when opening the screen on my P30... not sure what causes this

**Review by BMItr — APPROVED** [review]
tested on Nexus 5x and Samsung S20. looking good 👌 
PS: I also checked on a P20 Device, but was also working as expected (no crash). But I have no P30

**Review by axelherbstreith — APPROVED** [review]

**Comment by ralfgehrer** [comment]
> tested on Nexus 5x and Samsung S20. looking good 👌
> PS: I also checked on a P20 Device, but was also working as expected (no crash). But I have no P30

The crash was due to English language settings. This will updated with the upcoming Translation Delivery. @axelherbstreith  could validate it with German language settings.

**Pull request merged**